files/zh-cn/web/javascript/reference/global_objects/object/index.html

```diff
@@ -99,17 +89,48 @@ <h2 id="静态方法">静态方法</h2>
  <dd>返回给定对象自身可枚举值的数组。</dd>
 </dl>
 
-<h2 id="Object_实例和_Object_原型对象"><code>Object</code> 实例和 <code>Object</code> 原型对象</h2>
 
-<p>JavaScript中的所有对象都来自 <code>Object</code>；所有对象从{{jsxref("Object.prototype")}}继承方法和属性，尽管它们可能被覆盖。例如，其他构造函数的原型将覆盖 <code>constructor</code> 属性并提供自己的 <code>toString()</code> 方法。<code>Object</code> 原型对象的更改将传播到所有对象，除非受到这些更改的属性和方法将沿原型链进一步覆盖。</p>
+<h2 id="实例属性">实例属性</h2>
+
+<dl>
+  <dt>{{jsxref("Object.prototype.constructor")}}</dt>
+  <dd>一个引用值，指向 Object 构造函数</dd>
+  <dt>{{jsxref("Object.prototype.proto", "Object.prototype.__proto__")}}</dt>
+  <dd>指向一个对象，当一个 object 实例化时，使用该对象作为实例化对象的原型</dd>
+</dl>
 
-<h3 id="属性">属性</h3>
+<h2 id="实例属性">实例方法</h2>
 
-<div>{{ page('zh-CN/docs/JavaScript/Reference/Global_Objects/Object/prototype', 'Properties') }}</div>
+<dl>
+  <dt>{{jsxref("Object.prototype.__defineGetter__", "Object.prototype.__defineGetter__()")}}</dt>
+  <dd>一个属性值为函数的属性，当该属性被访问时，执行该函数，并且返回函数的返回值。</dd>
```

Comment:
> 一个属性值为函数的属性

This is wrong! The property's value is depend on what the getter function return, it does not mean that the property's value is a function (same as `defineSetter`). Please correct both.

> Associates a function with a property that

The point is "Associate"